libmodernize/main.py

```diff
@@ -63,6 +72,10 @@ def main(args=None):
                       "(only useful for Python 2.6+).")
     parser.add_option("--no-six", action="store_true", default=False,
                       help="Exclude fixes that depend on the six package.")
+    parser.add_option("-U", "--unix-line-endings", action="store_true", default=False,
+                      help="Write files with Unix (LF) line endings.")
+    parser.add_option("-W", "--windows-line-endings", action="store_true", default=False,
```

Comment:
I don't think these need short options, especially since we already have lowercase `-w`. Explicit is better than implicit.